app/src/main/java/com/hedvig/app/feature/crossselling/ui/CrossSellData.kt

```diff
@@ -69,3 +104,10 @@ data class CrossSellData(
         )
     }
 }
+
+fun CrossSellType.toCrossSellType() = when (this) {
+    CrossSellType.ACCIDENT -> CrossSellData.CrossSellType.ACCIDENT
```

Comment:
I think having us be explicit about the `CrossSellType` will erode some of the genericity of this solution - currently, any type of cross-sell is supported. If we enable `sealed class for enums` for the `CrossSellType`, we can simply match on the `rawValue`, and we never have to build in support client-side. 😄

Reply:
Good idea